File: Candlestick_chart/Candlestick_chart.py
import mplfinance as mpf
import pandas as pd

# Exemple fictif de données OHLC
data = pd.DataFrame({
    'Open': [100, 102, 105, 103],
    'High': [105, 107, 108, 106],
    'Low': [98, 101, 103, 102],
    'Close': [104, 106, 104, 105],
}, index=pd.date_range("2023-01-01", periods=4))

# Graphique en bougies japonaises
mpf.plot(data, type='candle', title="Graphique en Bougies Japonaises")

File: Candlestick_chart/Candlestick_chart.r
library(quantmod)

# Exemple fictif de données OHLC
data <- data.frame(
  Open = c(100, 102, 105, 103),
  High = c(105, 107, 108, 106),
  Low = c(98, 101, 103, 102),
  Close = c(104, 106, 104, 105)
)
xts_data <- xts(data, order.by = as.Date("2023-01-01") + 0:3)

# Graphique en bougies japonaises
candleChart(xts_data, type = "candles", main = "Graphique en Bougies Japonaises")
